cEP-0005.md

```diff
@@ -0,0 +1,157 @@
+# coala Configuration
```

Comment:
Huh I didn't realize the type thing. That is indeed useful (type checking, limits, ...). Static code analysis for a configuration file of a static code analysis tool!

JSON quite easy to load (no need for sooo much parsing that we have in coalib). If we're looking at supporting it, at least both must be supported (It's atleast very easy to detect if a config is JSON or ini).